web/src/main/java/se/inera/intyg/rehabstod/service/sjukfall/testability/TestabilityServiceImpl.java

```diff
@@ -19,20 +19,46 @@
 
 package se.inera.intyg.rehabstod.service.sjukfall.testability;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Service;
-import se.inera.intyg.rehabstod.integration.it.service.IntygstjanstRestIntegrationService;
+import org.springframework.web.client.RestTemplate;
+import se.inera.intyg.rehabstod.integration.it.dto.CreateSickLeaveRequestDTO;
+import se.inera.intyg.rehabstod.integration.it.dto.CreateSickLeaveResponseDTO;
+import se.inera.intyg.rehabstod.integration.it.dto.TestDataOptionsDTO;
 
 @Service
 public class TestabilityServiceImpl implements TestabilityService {
 
```

Comment:
Consider adding a @Profile annotation to avoid creation of service in non-test evironments?

Reply:
Good call!

Added @Profile("dev, testability-api"). Will this do the work?

Reply:
Yes it should do the job